src/graph/validator/AdminValidator.cpp

```diff
@@ -20,8 +20,8 @@ Status CreateSpaceValidator::validateImpl() {
   ifNotExist_ = sentence->isIfNotExist();
   auto status = Status::OK();
   spaceDesc_.set_space_name(std::move(*(sentence->spaceName())));
-  if (sentence->groupName()) {
-    spaceDesc_.set_group_name(std::move(*(sentence->groupName())));
+  if (sentence->zoneNames()) {
+    spaceDesc_.set_zone_names(std::move(sentence->zoneNames()->zoneNames()));
```

Comment:
why  `->zoneNames()->zoneNames()`? just curious.

Reply:
It's just typed conversion and return the std::vector<std::string>

Reply:
compile error by clang here. no need std::move